misc/loop/docker-compose.production.yml

```diff
@@ -0,0 +1,142 @@
+      - portal-loop
+    ports:
+      - "80:80"
+      - "443:443"
+      - "26657:26657"
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+      - ./traefik:/etc/traefik/configs
+      - ./traefik/letsencrypt:/letsencrypt
+
+  gnoweb:
+    image: ghcr.io/gnolang/gno/gnoweb-slim
+    restart: unless-stopped
+    env_file: ".env"
+    entrypoint:
+      - gnoweb
+      - --bind=0.0.0.0:8888
+      - --remote=traefik:26657
+      - --faucet-url=https://faucet-api.portal.gnoteam.com
+      - --captcha-site=$CAPTCHA_SITE_KEY
+      - --with-analytics
+      - --help-chainid=portal-loop
+      - --help-remote=https://rpc.gno.land:443
+    networks:
+      - portal-loop
+    labels:
+      com.centurylinklabs.watchtower.enable: "true"
+      traefik.enable: "true"
+      traefik.http.routers.gnoweb.entrypoints: "web,websecure"
+      traefik.http.routers.gnoweb.rule: "Host(`gno.land`) || Host(`portal.gnoteam.com`)"
+      traefik.http.routers.gnoweb.tls: "true"
+      traefik.http.routers.gnoweb.tls.certresolver: "le"
+
+  gnofaucet:
+    image: ghcr.io/gnolang/gno/gnofaucet-slim
+    networks:
+      - portal-loop
+    command:
+      - "serve"
+      - "--listen-address=0.0.0.0:5050"
+      - "--chain-id=portal-loop"
+      - "--is-behind-proxy=true"
+      - "--mnemonic=${FAUCET_MNEMONIC}"
+      - "--num-accounts=1"
+      - "--remote=http://traefik:26657"
+      - "--captcha-secret=${CAPTCHA_SECRET_KEY}"
+    env_file: ".env"
+    # environment:
+    # from .env
+    # - RECAPTCHA_SECRET_KEY
+    labels:
+      com.centurylinklabs.watchtower.enable: "true"
+      traefik.enable: "true"
+      traefik.http.routers.gnofaucet-api.entrypoints: "websecure"
+      # traefik.http.routers.gnofaucet-api.rule: "Host(`faucet-api.gno.land`) || Host(`faucet-api.portal.gnoteam.com`)"
+      traefik.http.routers.gnofaucet-api.rule: "Host(`faucet-api.portal.gnoteam.com`)"
+      traefik.http.routers.gnofaucet-api.tls: "true"
+      traefik.http.routers.gnofaucet-api.tls.certresolver: "le"
+      traefik.http.middlewares.gnofaucet-ratelimit.ratelimit.average: "6"
+      traefik.http.middlewares.gnofaucet-ratelimit.ratelimit.period: "1m"
+
+  portalloopd:
+    # image: ghcr.io/gnolang/gno/portalloopd
+    image: ghcr.io/albttx/gno/portalloopd
+    restart: unless-stopped
+    volumes:
+      - ./scripts:/scripts
+      - ./backups:/backups
+      - ./traefik:/etc/traefik/configs
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+    networks:
+      - portal-loop
+    ports:
+      - 127.0.0.1:9090:9090
+    environment:
+      HOST_PWD: $PWD
+      BACKUP_DIR: "/backups"
+      RPC_URL: "http://traefik:26657"
+      PROM_ADDR: "0.0.0.0:9090"
+      TRAEFIK_GNO_FILE: "/etc/traefik/configs/gno.yml"
+    extra_hosts:
+      - host.docker.internal:host-gateway
+    labels:
+      - "com.centurylinklabs.watchtower.enable=true"
+
+  autocounterd:
+    image: ghcr.io/albttx/gno/autocounterd
+    restart: unless-stopped
+    env_file: ".env"
```

Comment:
`.env` is already the default file for environment variables, so this is redundant.

I suggest removing these lines and adding a `.env.sample` file with the expected environment variables.